package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-browser-tools",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
There are no any changes in public API, so it's not a minor change. 
More than that I suppose we are not going to release a new version with only this enhancement. Maybe it's better to implement some more features for linux and then increase the version?